packages/cli-platform-ios/src/tools/selectFromInteractiveMode.ts

```diff
@@ -9,25 +9,25 @@ import {
 interface Args {
   scheme?: string;
   mode?: string;
-  projectInfo: IosProjectInfo;
+  info: IosInfo | undefined;
 }
 
 export async function selectFromInteractiveMode({
   scheme,
   mode,
-  projectInfo,
+  info,
 }: Args): Promise<{scheme?: string; mode?: string}> {
   let newScheme = scheme;
   let newMode = mode;
 
-  if (projectInfo.schemes.length > 1) {
-    newScheme = await promptForSchemeSelection(projectInfo);
+  if (info && info?.schemes && info.schemes.length > 1) {
+    newScheme = await promptForSchemeSelection(info.schemes);
   } else {
     logger.info(`Automatically selected ${chalk.bold(scheme)} scheme.`);
   }
 
-  if (projectInfo.configurations.length > 1) {
-    newMode = await promptForConfigurationSelection(projectInfo);
+  if (info && info?.configurations && info?.configurations?.length > 1) {
+    newMode = await promptForConfigurationSelection(info.configurations);
```

Comment:
Same here. Though in this case, you need to be careful about `info?.configurations?.length > 1` since it might return `undefined`, which cannot be compared with numbers. It's fine here because of the previous checks.
```suggestion
  const configurations = info?.configurations;
  if (configurations && configurations.length > 1) {
    newMode = await promptForConfigurationSelection(configurations);
```